packages/salesforcedx-vscode-core/package.json

```diff
@@ -25,7 +25,7 @@
     "Other"
   ],
   "dependencies": {
-    "@heroku/functions-core": "0.0.7",
+    "@heroku/functions-core": "0.1.4",
```

Comment:
Curious why package-lock.json is not being updated (I don't see it getting updated in my local as well).

Reply:
I ran `git clean -xfd` (and the also `npm install` and `npm run compile`) and 0.1.4 was pulled down.